test/docfx.Tests/docfx.Tests.csproj

```diff
@@ -26,6 +22,7 @@
     <ProjectReference Include="..\..\src\Microsoft.DocAsCode.DataContracts.Common\Microsoft.DocAsCode.DataContracts.Common.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.DocAsCode.DataContracts.ManagedReference\Microsoft.DocAsCode.DataContracts.ManagedReference.csproj" />
     <ProjectReference Include="..\..\src\docfx\docfx.csproj" />
+    <ProjectReference Include="..\Microsoft.DocAsCode.Tests.Common\Microsoft.DocAsCode.Tests.Common.csproj" />
```

Comment:
it's already in test folder...

---
In reply to: [119266335](https://github.com/dotnet/docfx/pull/1710#discussion_r119266335) [](ancestors = 119266335)